docs/portal/team-members.md

```diff
@@ -20,15 +20,25 @@ In the UID2 Portal, on the **Team Members** page, Admin users can perform all ac
 
 To access the Team Members page, go to the top right of the UID2 Portal user interface where your name is displayed. Click the down arrow and choose **Manage Team Members**.
 
+For each team member, the following information is displayed:
+- Name
+- Email
+- Job Function
+- Role
+- Actions: the available actions are **Edit** or **Delete**.
+
 For an illustration, see [UID2 Portal Structure](portal-overview.md#uid2-portal-structure).
 
 ## User Roles
 
-Each team member has one of two user roles: Admin or Operations. Admin users can perform all activities related to UID2 participation, as summarized in the [UID2 Portal Overview](portal-overview.md).
+Each team member has one of the roles shown in the following table.
 
-Users with the Operations role can perform most of the same actions as Admin users. However, they cannot add, modify, or remove team members. Instead, they have read-only access to the **Manage Team Members** page.
+| Role | Actions |
+| :--- | :--- |
+| Admin | Users with the Admin role can perform all activities related to UID2 participation, as summarized in the [UID2 Portal Overview](portal-overview.md), for the current participant organization. |
+| Operations | Users with the Operations role can perform most of the same actions as Admin users. However, they cannot add, modify, or remove team members. Instead, they have read-only access to the **Manage Team Members** page. |
```

Comment:
Operations also cannot view the Audit Trail if you want to add that here

Reply:
Done, thx.